Sources/Build/BuildPlan.swift

```diff
@@ -247,6 +260,8 @@ public final class ClangTargetBuildDescription {
             // Using modules currently conflicts with the Windows and Android SDKs.
             args += ["-fmodules", "-fmodule-name=" + target.c99name]
         }
+
+        args += ["-F", buildParameters.buildPath.pathString]
```

Comment:
can you make this conditional on presence of at least one framework?

Reply:
I did it this way to mimic Xcode, which always includes it, even when there are no frameworks to link against. Is there a reason they do it that way?

Reply:
Done in latest commit.